index.html

```diff
@@ -1969,39 +2033,53 @@ <h2>The <dfn>NDEFRecordType</dfn> string</h2>
     </p>
   </section> <!-- visibility & focus: the suspended state -->
 
-  <section><h3>Aborting pending push operation</h3>
+  <section><h3>Clearing the pending push tuple</h3>
   <p>
-    To attempt to <dfn>abort a pending push operation</dfn> on an
+    To <dfn>clear the pending push tuple</dfn> on an
     <a>environment settings object</a>, perform the following steps:
     <ol class=algorithm>
       <li>
         If there is no <a>pending push tuple</a> |tuple|, abort these steps.
       </li>
       <li>
-        Let |writer:NFCWriter| be |tuple|'s writer.
+        Stop the <a>pending push timer</a> if active.
       </li>
       <li>
-        Let |p:Promise| be |tuple|'s promise.
+        Stop the <a>pending push handler</a> if running.
+      </li>
+      <li>
+        Clear the <a>pending push tuple</a>.
```

Comment:
It's a bit weird that these internal slots is per NFCWriter object but you treat them as global... What if there are multiple NFCWriter objects

Reply:
Yes, they are per NFCWriter.
The named "pending ..." refer to the internal slots of the one and only "pending push tuple", as the text says. 
Alternatively they could be put in the "pending push tuple", but then it's weird to have the writer object there and some free hanging internal slots of the writer object.

Reply:
Then we should refer to the object itself and not to prose like 'pending push tuple' ie. |writer|.[[\PushTuple]] etc (don't remember what name you used)

Reply:
It doesn't seem to be per NFCWriter object as the writer is part of the tuple'.... Hard to see this on phone